migration/1663594895751-takePowerSnapshotProcedure.ts

```diff
@@ -1,7 +1,7 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
 // tslint:disable-next-line:class-name
-export class TakePowerBoostingSnapshotProcedure1663594895750
+export class TakePowerBoostingSnapshotProcedure1663594895751
```

Comment:
I changed the name to make sure it will run again although it already has been executed

Reply:
Got it!